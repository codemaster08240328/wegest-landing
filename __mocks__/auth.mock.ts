/**
 * Email: testusergmail.com
 * password: testuser123
 */

export const signin = async ({
  email,
  password,
}: {
  email: string
  password: string
}) => {
  return new Promise((resolve) => {
    setTimeout(() => {
      resolve(1)
    }, 200)
  }).then(() => {
    if (email === 'testuser@gmail.com' && password === 'testuser123') {
      return {
        token: 'auth.token',
        user: {
          email: 'testuser@gmail.com',
          firstName: 'James',
          lastName: 'Wo',
          gender: 'Male',
          phone: 'xxxxx',
          role: 'user',
        },
      }
    }

    return 'invalid'
  })
}
