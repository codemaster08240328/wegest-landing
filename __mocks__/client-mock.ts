export const clientUnderProcessing = [
  {
    id: '224FDF9255E5F99900',
    firstName: 'Giulia',
    lastName: 'Malu',
    gender: 'Female',
    frequency: 'not classified',
    frequencyVariation: '=',
    techNotes: [
      {
        date: 'April 11, 2021 - 12:23',
        notes: [
          {
            id: '564',
            type: 'Nail Varnish',
            note: 'CCCCCCCC',
          },
        ],
      },
    ],
    serviceAssigned: [
      {
        id: '4038DF9058',
        name: 'California massage',
        operator: true,
        operatorId: 'LUC',
      },
      {
        id: '4638DF93',
        name: 'Crease',
        operator: true,
        operatorId: 'LUC',
      },
      {
        id: '4638DB94',
        name: 'Cut',
        operator: true,
        operatorId: 'LUC',
      },
      {
        id: '4638DB9B',
        name: 'Full color',
        operator: true,
        operatorId: 'LUC',
      },
    ],
    serviceHistory: [
      {
        id: '4638DB94',
        name: 'Cut',
        days: 68,
      },
      {
        id: '4638DB9B',
        name: 'Full color',
        days: 68,
      },
      {
        id: '4638DF93',
        name: 'Crease',
        days: 66,
      },
    ],
    productAssigned: [
      {
        id: '423FDB9451E5',
        name: 'bbbbprova',
        operator: true,
        operatorId: 'LUC',
      },
    ],
  },
  {
    id: '224FDD9B56E2F19701',
    firstName: 'Sabina',
    lastName: 'Abdalova',
    gender: 'Female',
    frequency: 'monthly',
    frequencyVariation: '-',
    techNotes: [
      {
        date: 'April 12, 2021 - 09:38',
        notes: [
          {
            id: '7',
            type: 'General',
            note: 'AAAAAAAA',
          },
          {
            id: '508',
            type: 'Cut',
            note: 'BBBBBBBB',
          },
          {
            id: '344',
            type: 'Spikes',
            note: 'HHHHHHH',
          },
          {
            id: '324',
            type: 'Shampoo',
            note: 'SSSSSSS',
          },
        ],
      },
      {
        date: 'April 11, 2021 - 12:23',
        notes: [
          {
            id: '564',
            type: 'Nail Varnish',
            note: 'CCCCCCCC',
          },
        ],
      },
    ],
    serviceAssigned: [
      {
        id: '4038DF9058',
        name: 'California massage',
        operator: true,
        operatorId: 'LUC',
      },
      {
        id: '4638DF93',
        name: 'Crease',
        operator: true,
        operatorId: 'LUC',
      },
      {
        id: '4638DB94',
        name: 'Cut',
        operator: true,
        operatorId: 'LUC',
      },
      {
        id: '4638DB9B',
        name: 'Full color',
        operator: true,
        operatorId: 'LUC',
      },
    ],
    serviceHistory: [
      {
        id: '4638DB94',
        name: 'Cut',
        days: 68,
      },
      {
        id: '4638DB9B',
        name: 'Full color',
        days: 68,
      },
      {
        id: '4638DF93',
        name: 'Crease',
        days: 66,
      },
    ],
    productAssigned: [
      {
        id: '423FDB9451E5',
        name: 'bbbbprova',
        operator: true,
        operatorId: 'LUC',
      },
    ],
  },
  {
    id: '224FDD9A56E6F49500',
    firstName: 'Bahar',
    lastName: 'Abbaspour',
    gender: 'Female',
    frequency: 'monthly',
    frequencyVariation: '-',
    techNotes: [
      {
        date: 'April 11, 2021 - 12:23',
        notes: [
          {
            id: '564',
            type: 'Nail Varnish',
            note: 'CCCCCCCC',
          },
        ],
      },
    ],
    serviceAssigned: [
      {
        id: '4038DF9058',
        name: 'California massage',
        operator: true,
        operatorId: 'LUC',
      },
      {
        id: '4638DF93',
        name: 'Crease',
        operator: true,
        operatorId: 'LUC',
      },
      {
        id: '4638DB94',
        name: 'Cut',
        operator: true,
        operatorId: 'LUC',
      },
      {
        id: '4638DB9B',
        name: 'Full color',
        operator: true,
        operatorId: 'LUC',
      },
    ],
    serviceHistory: [
      {
        id: '4638DB94',
        name: 'Cut',
        days: 68,
      },
      {
        id: '4638DB9B',
        name: 'Full color',
        days: 68,
      },
      {
        id: '4638DF93',
        name: 'Crease',
        days: 66,
      },
    ],
    productAssigned: [
      {
        id: '423FDB9451E5',
        name: 'bbbbprova',
        operator: true,
        operatorId: 'LUC',
      },
    ],
  },
  {
    id: '224FDD9759E8F89206',
    firstName: 'Abbate',
    lastName: 'Abballati',
    gender: 'Male',
    frequency: 'fortnightly',
    frequencyVariation: '-',
    techNotes: [
      {
        date: 'April 11, 2021 - 12:23',
        notes: [
          {
            id: '564',
            type: 'Nail Varnish',
            note: 'CCCCCCCC',
          },
        ],
      },
    ],
    serviceAssigned: [
      {
        id: '4038DF9058',
        name: 'California massage',
        operator: true,
        operatorId: 'LUC',
      },
      {
        id: '4638DF93',
        name: 'Crease',
        operator: true,
        operatorId: 'LUC',
      },
      {
        id: '4638DB94',
        name: 'Cut',
        operator: true,
        operatorId: 'LUC',
      },
      {
        id: '4638DB9B',
        name: 'Full color',
        operator: true,
        operatorId: 'LUC',
      },
    ],
    serviceHistory: [
      {
        id: '4638DB94',
        name: 'Cut',
        days: 68,
      },
      {
        id: '4638DB9B',
        name: 'Full color',
        days: 68,
      },
      {
        id: '4638DF93',
        name: 'Crease',
        days: 66,
      },
    ],
    productAssigned: [
      {
        id: '423FDB9451E5',
        name: 'bbbbprova',
        operator: true,
        operatorId: 'LUC',
      },
    ],
  },
  {
    id: '224FDD9658E4F3970D',
    firstName: 'Mobina',
    lastName: 'Abs',
    gender: 'Female',
    frequency: 'quarterly',
    frequencyVariation: '-',
    techNotes: [
      {
        date: 'April 11, 2021 - 12:23',
        notes: [
          {
            id: '564',
            type: 'Nail Varnish',
            note: 'CCCCCCCC',
          },
        ],
      },
    ],
    serviceAssigned: [
      {
        id: '4038DF9058',
        name: 'California massage',
        operator: true,
        operatorId: 'LUC',
      },
      {
        id: '4638DF93',
        name: 'Crease',
        operator: true,
        operatorId: 'LUC',
      },
      {
        id: '4638DB94',
        name: 'Cut',
        operator: true,
        operatorId: 'LUC',
      },
      {
        id: '4638DB9B',
        name: 'Full color',
        operator: true,
        operatorId: 'LUC',
      },
    ],
    serviceHistory: [
      {
        id: '4638DB94',
        name: 'Cut',
        days: 68,
      },
      {
        id: '4638DB9B',
        name: 'Full color',
        days: 68,
      },
      {
        id: '4638DF93',
        name: 'Crease',
        days: 66,
      },
    ],
    productAssigned: [
      {
        id: '423FDB9451E5',
        name: 'bbbbprova',
        operator: true,
        operatorId: 'LUC',
      },
    ],
  },
  {
    id: '224FDD9752E1F49904',
    firstName: 'Cristina',
    lastName: 'Abdulla',
    gender: 'Female',
    frequency: 'fortnightly',
    frequencyVariation: '-',
    techNotes: [
      {
        date: 'April 11, 2021 - 12:23',
        notes: [
          {
            id: '564',
            type: 'Nail Varnish',
            note: 'CCCCCCCC',
          },
        ],
      },
    ],
    serviceAssigned: [
      {
        id: '4038DF9058',
        name: 'California massage',
        operator: true,
        operatorId: 'LUC',
      },
      {
        id: '4638DF93',
        name: 'Crease',
        operator: true,
        operatorId: 'LUC',
      },
      {
        id: '4638DB94',
        name: 'Cut',
        operator: true,
        operatorId: 'LUC',
      },
      {
        id: '4638DB9B',
        name: 'Full color',
        operator: true,
        operatorId: 'LUC',
      },
    ],
    serviceHistory: [
      {
        id: '4638DB94',
        name: 'Cut',
        days: 68,
      },
      {
        id: '4638DB9B',
        name: 'Full color',
        days: 68,
      },
      {
        id: '4638DF93',
        name: 'Crease',
        days: 66,
      },
    ],
    productAssigned: [
      {
        id: '423FDB9451E5',
        name: 'bbbbprova',
        operator: true,
        operatorId: 'LUC',
      },
    ],
  },
  {
    id: '224FDE9255E1F79907',
    firstName: 'Erlina',
    lastName: 'Abaring',
    gender: 'Female',
    frequency: 'weekly',
    frequencyVariation: '=',
    techNotes: [
      {
        date: 'April 11, 2021 - 12:23',
        notes: [
          {
            id: '564',
            type: 'Nail Varnish',
            note: 'CCCCCCCC',
          },
        ],
      },
    ],
    serviceAssigned: [
      {
        id: '4038DF9058',
        name: 'California massage',
        operator: true,
        operatorId: 'LUC',
      },
      {
        id: '4638DF93',
        name: 'Crease',
        operator: true,
        operatorId: 'LUC',
      },
      {
        id: '4638DB94',
        name: 'Cut',
        operator: true,
        operatorId: 'LUC',
      },
      {
        id: '4638DB9B',
        name: 'Full color',
        operator: true,
        operatorId: 'LUC',
      },
    ],
    serviceHistory: [
      {
        id: '4638DB94',
        name: 'Cut',
        days: 68,
      },
      {
        id: '4638DB9B',
        name: 'Full color',
        days: 68,
      },
      {
        id: '4638DF93',
        name: 'Crease',
        days: 66,
      },
    ],
    productAssigned: [
      {
        id: '423FDB9451E5',
        name: 'bbbbprova',
        operator: true,
        operatorId: 'LUC',
      },
    ],
  },
  {
    id: '224FDE9458E3F59601',
    firstName: 'A',
    lastName: 'A',
    gender: 'Female',
    frequency: 'quarterly',
    frequencyVariation: '-',
    techNotes: [
      {
        date: 'April 11, 2021 - 12:23',
        notes: [
          {
            id: '564',
            type: 'Nail Varnish',
            note: 'CCCCCCCC',
          },
        ],
      },
    ],
    serviceAssigned: [
      {
        id: '4038DF9058',
        name: 'California massage',
        operator: true,
        operatorId: 'LUC',
      },
      {
        id: '4638DF93',
        name: 'Crease',
        operator: true,
        operatorId: 'LUC',
      },
      {
        id: '4638DB94',
        name: 'Cut',
        operator: true,
        operatorId: 'LUC',
      },
      {
        id: '4638DB9B',
        name: 'Full color',
        operator: true,
        operatorId: 'LUC',
      },
    ],
    serviceHistory: [
      {
        id: '4638DB94',
        name: 'Cut',
        days: 68,
      },
      {
        id: '4638DB9B',
        name: 'Full color',
        days: 68,
      },
      {
        id: '4638DF93',
        name: 'Crease',
        days: 66,
      },
    ],
    productAssigned: [
      {
        id: '423FDB9451E5',
        name: 'bbbbprova',
        operator: true,
        operatorId: 'LUC',
      },
    ],
  },
  {
    id: '224FDB9053E6F090',
    firstName: 'Marco',
    lastName: 'Pernando',
    gender: 'Male',
    frequency: 'quarterly',
    frequencyVariation: '-',
    techNotes: [
      {
        date: 'April 11, 2021 - 12:23',
        notes: [
          {
            id: '564',
            type: 'Nail Varnish',
            note: 'CCCCCCCC',
          },
        ],
      },
    ],
    serviceAssigned: [
      {
        id: '4038DF9058',
        name: 'California massage',
        operator: true,
        operatorId: 'LUC',
      },
      {
        id: '4638DF93',
        name: 'Crease',
        operator: true,
        operatorId: 'LUC',
      },
      {
        id: '4638DB94',
        name: 'Cut',
        operator: true,
        operatorId: 'LUC',
      },
      {
        id: '4638DB9B',
        name: 'Full color',
        operator: true,
        operatorId: 'LUC',
      },
    ],
    serviceHistory: [
      {
        id: '4638DB94',
        name: 'Cut',
        days: 68,
      },
      {
        id: '4638DB9B',
        name: 'Full color',
        days: 68,
      },
      {
        id: '4638DF93',
        name: 'Crease',
        days: 66,
      },
    ],
    productAssigned: [
      {
        id: '423FDB9451E5',
        name: 'bbbbprova',
        operator: true,
        operatorId: 'LUC',
      },
    ],
  },
]

export const clientInPersonalData = [
  {
    id: '224FDB9053E6F090',
    firstName: 'Marco',
    lastName: 'Pernando',
    gender: 'Male',
  },
  {
    id: '224FDE9458E3F59601',
    firstName: 'A',
    lastName: 'A',
    gender: 'Female',
  },
  {
    id: '224FDE9A55E1F79004',
    firstName: 'BB',
    lastName: 'AA',
    gender: 'Male',
  },
  {
    id: '224FDE9557E1F29407',
    firstName: 'AChi',
    lastName: 'AAA',
    gender: 'Female',
  },
  {
    id: '224FDE9255E1F79907',
    firstName: 'Erlina',
    lastName: 'Abaring',
    gender: 'Female',
  },
  {
    id: '224FDD9759E8F89206',
    firstName: 'Abbate',
    lastName: 'Abballati',
    gender: 'Male',
  },
  {
    id: '224FDE9A55E1F2970C',
    firstName: 'Lorenzo',
    lastName: 'Abbandonato',
    gender: 'Male',
  },
  {
    id: '224FDD9A56E6F49500',
    firstName: 'Bahar',
    lastName: 'Abbaspour',
    gender: 'Female',
  },
  {
    id: '224FDF9352E9F19303',
    firstName: 'GAETANO',
    lastName: 'ABBATE',
    gender: 'Female',
  },
  {
    id: '224FDD9B56E2F19701',
    firstName: 'Sabina',
    lastName: 'Abdalova',
    gender: 'Female',
  },
  {
    id: '224FDD9752E1F49904',
    firstName: 'Cristina',
    lastName: 'Abdulla',
    gender: 'Female',
  },
  {
    id: '224FDD9652E0F1920C',
    firstName: 'Nazanin',
    lastName: 'Abedi',
    gender: 'Female',
  },
  {
    id: '224FDF9352E6F69202',
    firstName: 'Nana',
    lastName: 'Abena',
    gender: 'Female',
  },
  {
    id: '224FDD9B56E6F69904',
    firstName: 'Rakim',
    lastName: 'Abid',
    gender: 'Female',
  },
  {
    id: '224FDD9658E4F3970D',
    firstName: 'Mobina',
    lastName: 'Abs',
    gender: 'Female',
  },
  {
    id: '224FDE9152E5F8940D',
    firstName: 'Raeed',
    lastName: 'Abualrod',
    gender: 'Female',
  },
]

export const TechTypologies = [
  {
    id: '7',
    type: 'General',
    note: '',
  },
  {
    id: '508',
    type: 'Skin',
    note: '',
  },
  {
    id: '510',
    type: 'Spikes',
    note: '',
  },
  {
    id: '513',
    type: 'Shampoo',
    note: '',
  },
  {
    id: '564',
    type: 'Nail Varnish',
    note: '',
  },
  {
    id: '548',
    type: 'Treatments',
    note: '',
  },
]
