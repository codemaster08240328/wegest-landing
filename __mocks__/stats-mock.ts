import { IResume } from '@/shared/@types/models/stastics'

export const resume: Record<string, IResume> = {
  month: {
    revenue: 4675.64,
    avgReceiptValue: 203.29,
    fees: 3878.77,
    serviceFees: 3806.25,
    productFees: 72.52,
    attendance: 23,
    newCustomers: 1,
    handledCustomer: 11,
    expenses: 1105.32,
    netIncome: 3570.32,
  },

  today: {
    revenue: 0,
    avgReceiptValue: 0,
    fees: 0,
    serviceFees: 0,
    productFees: 0,
    attendance: 0,
    newCustomers: 0,
    handledCustomer: 0,
    expenses: 0,
    netIncome: 0,
  },

  yesterday: {
    revenue: 166,
    avgReceiptValue: 166,
    fees: 0,
    serviceFees: 0,
    productFees: 0,
    attendance: 1,
    newCustomers: 0,
    handledCustomer: 1,
    expenses: 0,
    netIncome: 166,
  },
}
