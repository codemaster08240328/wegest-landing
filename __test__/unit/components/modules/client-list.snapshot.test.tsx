import renderer from 'react-test-renderer'
import ClientList from '@/components/modules/client-list'
import { clientUnderProcessing } from '@/__mocks__/client-mock'

describe('<ClientList> snapshot', () => {
  it('renders correctly with props', () => {
    expect(
      renderer.create(<ClientList clients={clientUnderProcessing} />).toJSON(),
    ).toMatchSnapshot()
  })
})
