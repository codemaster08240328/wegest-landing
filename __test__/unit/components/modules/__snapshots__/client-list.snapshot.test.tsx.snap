// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ClientList> snapshot renders correctly with props 1`] = `
Array [
  <div
    className="MuiBox-root MuiBox-root-4 makeStyles-root-1 "
  >
    <div
      className="MuiBox-root MuiBox-root-5 makeStyles-avatar-2"
    >
      <div
        bgcolor="#f06292"
        className="MuiAvatar-root Component-root-6 Component-root-8 MuiAvatar-circle MuiAvatar-colorDefault Component-colorDefault-7 Component-colorDefault-9"
        data-testid="el-avatar"
        size={50}
      >
        GM
      </div>
      Giulia Malu
    </div>
    <button
      className="MuiButtonBase-root MuiIconButton-root Component-root-10 Component-root-11 makeStyles-button-3"
      data-testid="el-icon-button"
      disabled={false}
      onBlur={[Function]}
      onClick={[Function]}
      onDragLeave={[Function]}
      onFocus={[Function]}
      onKeyDown={[Function]}
      onKeyUp={[Function]}
      onMouseDown={[Function]}
      onMouseLeave={[Function]}
      onMouseUp={[Function]}
      onTouchEnd={[Function]}
      onTouchMove={[Function]}
      onTouchStart={[Function]}
      tabIndex={0}
      type="button"
    >
      <span
        className="MuiIconButton-label"
      >
        <span
          aria-hidden={true}
          className="material-icons MuiIcon-root"
          data-testid="el-icon"
        >
          <span
            aria-hidden={true}
            className="material-icons MuiIcon-root material-icons-outlined"
            data-testid="el-icon"
          >
            info
          </span>
        </span>
      </span>
    </button>
  </div>,
  <div
    className="MuiBox-root MuiBox-root-12 makeStyles-root-1 "
  >
    <div
      className="MuiBox-root MuiBox-root-13 makeStyles-avatar-2"
    >
      <div
        bgcolor="#f06292"
        className="MuiAvatar-root Component-root-6 Component-root-14 MuiAvatar-circle MuiAvatar-colorDefault Component-colorDefault-7 Component-colorDefault-15"
        data-testid="el-avatar"
        size={50}
      >
        SA
      </div>
      Sabina Abdalova
    </div>
    <button
      className="MuiButtonBase-root MuiIconButton-root Component-root-10 Component-root-16 makeStyles-button-3"
      data-testid="el-icon-button"
      disabled={false}
      onBlur={[Function]}
      onClick={[Function]}
      onDragLeave={[Function]}
      onFocus={[Function]}
      onKeyDown={[Function]}
      onKeyUp={[Function]}
      onMouseDown={[Function]}
      onMouseLeave={[Function]}
      onMouseUp={[Function]}
      onTouchEnd={[Function]}
      onTouchMove={[Function]}
      onTouchStart={[Function]}
      tabIndex={0}
      type="button"
    >
      <span
        className="MuiIconButton-label"
      >
        <span
          aria-hidden={true}
          className="material-icons MuiIcon-root"
          data-testid="el-icon"
        >
          <span
            aria-hidden={true}
            className="material-icons MuiIcon-root material-icons-outlined"
            data-testid="el-icon"
          >
            info
          </span>
        </span>
      </span>
    </button>
  </div>,
  <div
    className="MuiBox-root MuiBox-root-17 makeStyles-root-1 "
  >
    <div
      className="MuiBox-root MuiBox-root-18 makeStyles-avatar-2"
    >
      <div
        bgcolor="#f06292"
        className="MuiAvatar-root Component-root-6 Component-root-19 MuiAvatar-circle MuiAvatar-colorDefault Component-colorDefault-7 Component-colorDefault-20"
        data-testid="el-avatar"
        size={50}
      >
        BA
      </div>
      Bahar Abbaspour
    </div>
    <button
      className="MuiButtonBase-root MuiIconButton-root Component-root-10 Component-root-21 makeStyles-button-3"
      data-testid="el-icon-button"
      disabled={false}
      onBlur={[Function]}
      onClick={[Function]}
      onDragLeave={[Function]}
      onFocus={[Function]}
      onKeyDown={[Function]}
      onKeyUp={[Function]}
      onMouseDown={[Function]}
      onMouseLeave={[Function]}
      onMouseUp={[Function]}
      onTouchEnd={[Function]}
      onTouchMove={[Function]}
      onTouchStart={[Function]}
      tabIndex={0}
      type="button"
    >
      <span
        className="MuiIconButton-label"
      >
        <span
          aria-hidden={true}
          className="material-icons MuiIcon-root"
          data-testid="el-icon"
        >
          <span
            aria-hidden={true}
            className="material-icons MuiIcon-root material-icons-outlined"
            data-testid="el-icon"
          >
            info
          </span>
        </span>
      </span>
    </button>
  </div>,
  <div
    className="MuiBox-root MuiBox-root-22 makeStyles-root-1 "
  >
    <div
      className="MuiBox-root MuiBox-root-23 makeStyles-avatar-2"
    >
      <div
        bgcolor="#64b5f6"
        className="MuiAvatar-root Component-root-6 Component-root-24 MuiAvatar-circle MuiAvatar-colorDefault Component-colorDefault-7 Component-colorDefault-25"
        data-testid="el-avatar"
        size={50}
      >
        AA
      </div>
      Abbate Abballati
    </div>
    <button
      className="MuiButtonBase-root MuiIconButton-root Component-root-10 Component-root-26 makeStyles-button-3"
      data-testid="el-icon-button"
      disabled={false}
      onBlur={[Function]}
      onClick={[Function]}
      onDragLeave={[Function]}
      onFocus={[Function]}
      onKeyDown={[Function]}
      onKeyUp={[Function]}
      onMouseDown={[Function]}
      onMouseLeave={[Function]}
      onMouseUp={[Function]}
      onTouchEnd={[Function]}
      onTouchMove={[Function]}
      onTouchStart={[Function]}
      tabIndex={0}
      type="button"
    >
      <span
        className="MuiIconButton-label"
      >
        <span
          aria-hidden={true}
          className="material-icons MuiIcon-root"
          data-testid="el-icon"
        >
          <span
            aria-hidden={true}
            className="material-icons MuiIcon-root material-icons-outlined"
            data-testid="el-icon"
          >
            info
          </span>
        </span>
      </span>
    </button>
  </div>,
  <div
    className="MuiBox-root MuiBox-root-27 makeStyles-root-1 "
  >
    <div
      className="MuiBox-root MuiBox-root-28 makeStyles-avatar-2"
    >
      <div
        bgcolor="#f06292"
        className="MuiAvatar-root Component-root-6 Component-root-29 MuiAvatar-circle MuiAvatar-colorDefault Component-colorDefault-7 Component-colorDefault-30"
        data-testid="el-avatar"
        size={50}
      >
        MA
      </div>
      Mobina Abs
    </div>
    <button
      className="MuiButtonBase-root MuiIconButton-root Component-root-10 Component-root-31 makeStyles-button-3"
      data-testid="el-icon-button"
      disabled={false}
      onBlur={[Function]}
      onClick={[Function]}
      onDragLeave={[Function]}
      onFocus={[Function]}
      onKeyDown={[Function]}
      onKeyUp={[Function]}
      onMouseDown={[Function]}
      onMouseLeave={[Function]}
      onMouseUp={[Function]}
      onTouchEnd={[Function]}
      onTouchMove={[Function]}
      onTouchStart={[Function]}
      tabIndex={0}
      type="button"
    >
      <span
        className="MuiIconButton-label"
      >
        <span
          aria-hidden={true}
          className="material-icons MuiIcon-root"
          data-testid="el-icon"
        >
          <span
            aria-hidden={true}
            className="material-icons MuiIcon-root material-icons-outlined"
            data-testid="el-icon"
          >
            info
          </span>
        </span>
      </span>
    </button>
  </div>,
  <div
    className="MuiBox-root MuiBox-root-32 makeStyles-root-1 "
  >
    <div
      className="MuiBox-root MuiBox-root-33 makeStyles-avatar-2"
    >
      <div
        bgcolor="#f06292"
        className="MuiAvatar-root Component-root-6 Component-root-34 MuiAvatar-circle MuiAvatar-colorDefault Component-colorDefault-7 Component-colorDefault-35"
        data-testid="el-avatar"
        size={50}
      >
        CA
      </div>
      Cristina Abdulla
    </div>
    <button
      className="MuiButtonBase-root MuiIconButton-root Component-root-10 Component-root-36 makeStyles-button-3"
      data-testid="el-icon-button"
      disabled={false}
      onBlur={[Function]}
      onClick={[Function]}
      onDragLeave={[Function]}
      onFocus={[Function]}
      onKeyDown={[Function]}
      onKeyUp={[Function]}
      onMouseDown={[Function]}
      onMouseLeave={[Function]}
      onMouseUp={[Function]}
      onTouchEnd={[Function]}
      onTouchMove={[Function]}
      onTouchStart={[Function]}
      tabIndex={0}
      type="button"
    >
      <span
        className="MuiIconButton-label"
      >
        <span
          aria-hidden={true}
          className="material-icons MuiIcon-root"
          data-testid="el-icon"
        >
          <span
            aria-hidden={true}
            className="material-icons MuiIcon-root material-icons-outlined"
            data-testid="el-icon"
          >
            info
          </span>
        </span>
      </span>
    </button>
  </div>,
  <div
    className="MuiBox-root MuiBox-root-37 makeStyles-root-1 "
  >
    <div
      className="MuiBox-root MuiBox-root-38 makeStyles-avatar-2"
    >
      <div
        bgcolor="#f06292"
        className="MuiAvatar-root Component-root-6 Component-root-39 MuiAvatar-circle MuiAvatar-colorDefault Component-colorDefault-7 Component-colorDefault-40"
        data-testid="el-avatar"
        size={50}
      >
        EA
      </div>
      Erlina Abaring
    </div>
    <button
      className="MuiButtonBase-root MuiIconButton-root Component-root-10 Component-root-41 makeStyles-button-3"
      data-testid="el-icon-button"
      disabled={false}
      onBlur={[Function]}
      onClick={[Function]}
      onDragLeave={[Function]}
      onFocus={[Function]}
      onKeyDown={[Function]}
      onKeyUp={[Function]}
      onMouseDown={[Function]}
      onMouseLeave={[Function]}
      onMouseUp={[Function]}
      onTouchEnd={[Function]}
      onTouchMove={[Function]}
      onTouchStart={[Function]}
      tabIndex={0}
      type="button"
    >
      <span
        className="MuiIconButton-label"
      >
        <span
          aria-hidden={true}
          className="material-icons MuiIcon-root"
          data-testid="el-icon"
        >
          <span
            aria-hidden={true}
            className="material-icons MuiIcon-root material-icons-outlined"
            data-testid="el-icon"
          >
            info
          </span>
        </span>
      </span>
    </button>
  </div>,
  <div
    className="MuiBox-root MuiBox-root-42 makeStyles-root-1 "
  >
    <div
      className="MuiBox-root MuiBox-root-43 makeStyles-avatar-2"
    >
      <div
        bgcolor="#f06292"
        className="MuiAvatar-root Component-root-6 Component-root-44 MuiAvatar-circle MuiAvatar-colorDefault Component-colorDefault-7 Component-colorDefault-45"
        data-testid="el-avatar"
        size={50}
      >
        AA
      </div>
      A A
    </div>
    <button
      className="MuiButtonBase-root MuiIconButton-root Component-root-10 Component-root-46 makeStyles-button-3"
      data-testid="el-icon-button"
      disabled={false}
      onBlur={[Function]}
      onClick={[Function]}
      onDragLeave={[Function]}
      onFocus={[Function]}
      onKeyDown={[Function]}
      onKeyUp={[Function]}
      onMouseDown={[Function]}
      onMouseLeave={[Function]}
      onMouseUp={[Function]}
      onTouchEnd={[Function]}
      onTouchMove={[Function]}
      onTouchStart={[Function]}
      tabIndex={0}
      type="button"
    >
      <span
        className="MuiIconButton-label"
      >
        <span
          aria-hidden={true}
          className="material-icons MuiIcon-root"
          data-testid="el-icon"
        >
          <span
            aria-hidden={true}
            className="material-icons MuiIcon-root material-icons-outlined"
            data-testid="el-icon"
          >
            info
          </span>
        </span>
      </span>
    </button>
  </div>,
  <div
    className="MuiBox-root MuiBox-root-47 makeStyles-root-1 "
  >
    <div
      className="MuiBox-root MuiBox-root-48 makeStyles-avatar-2"
    >
      <div
        bgcolor="#64b5f6"
        className="MuiAvatar-root Component-root-6 Component-root-49 MuiAvatar-circle MuiAvatar-colorDefault Component-colorDefault-7 Component-colorDefault-50"
        data-testid="el-avatar"
        size={50}
      >
        MP
      </div>
      Marco Pernando
    </div>
    <button
      className="MuiButtonBase-root MuiIconButton-root Component-root-10 Component-root-51 makeStyles-button-3"
      data-testid="el-icon-button"
      disabled={false}
      onBlur={[Function]}
      onClick={[Function]}
      onDragLeave={[Function]}
      onFocus={[Function]}
      onKeyDown={[Function]}
      onKeyUp={[Function]}
      onMouseDown={[Function]}
      onMouseLeave={[Function]}
      onMouseUp={[Function]}
      onTouchEnd={[Function]}
      onTouchMove={[Function]}
      onTouchStart={[Function]}
      tabIndex={0}
      type="button"
    >
      <span
        className="MuiIconButton-label"
      >
        <span
          aria-hidden={true}
          className="material-icons MuiIcon-root"
          data-testid="el-icon"
        >
          <span
            aria-hidden={true}
            className="material-icons MuiIcon-root material-icons-outlined"
            data-testid="el-icon"
          >
            info
          </span>
        </span>
      </span>
    </button>
  </div>,
]
`;
